db/migrate/20221003153000_add_ansible_callback_enabled_to_templates.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddAnsibleCallbackEnabledToTemplates < ActiveRecord::Migration[6.0]
+  def change
+    add_column :templates, :ansible_callback_enabled, :boolean, default: false
+    Template.where(name: 'Ansible Roles - Ansible Default').update_all(ansible_callback_enabled: true)
```

Comment:
@adamruzicka did you mean doing this:
` Template.find(RemoteExecutionFeature.where(label: 'ansible_run_host').first.job_template_id).update_all(ansible_callback_enabled: true)`?

Reply:
Almost. The way the database schema is laid out, it is possible that one feature will have more than one template assigned to it.